### DISCUSSION TITLE
Please reopen #35690

### DISCUSSION THREAD

**Discussion by ioweb-gr** [PrestaShop/PrestaShop#35782] [Q&A] [answered]
Hi,

I've added plenty of information to the issue #35690 that should have never been closed as it points to a huge issue in the architecture of the cart to order conversion process. Please reopen it as it seems no one is looking at issues once they're closed

**Comment by ioweb-gr** [comment]
Anyone here to re-open the issue?

**Reply by kpodemski** [reply, marked as answer]
Hello @ioweb-gr 

I shared this issue with maintainers from PrestaShop SA to see if we can prioritize it. In the meantime, make sure to check PR I mentioned to see if it helps with the issue.
